src/Template/Layout/js/app/components/property-view/property-view.js

```diff
@@ -131,5 +149,28 @@ export default {
                 this.$refs.relation.loadRelatedObjects();
             }
         },
+        async loadInfoUsers() {
+            const creatorId = this.object?.meta?.created_by;
+            const modifierId = this.object?.meta?.modified_by;
+            let usersId = [creatorId, modifierId];
```

Comment:
```suggestion
            const usersId = [creatorId, modifierId];
```
chore fix: use const